src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -252,6 +254,14 @@ def verify_user_emails(
             results[user_id] = RpcVerifyUserEmail(email=email, exists=exists)
         return results
 
+    def get_user_avatar(self, *, user_id: int) -> RpcAvatar | None:
+        possible_avatar = UserAvatar.objects.filter(user_id=user_id).first()
```

Comment:
Should this be a `get_from_cache` instead of a filter query?

Reply:
You could use `get_from_cache`, but we would need to update the `UserAvatar` model as well. Currently it only supports cached reads on the `user` relation, not on `user_id`.